setup.py

```diff
@@ -52,19 +67,45 @@ def main():
     class SetupStepFailedError(ValueError):
         pass
 
-    def getInputPath(prompt):
-        import shlex
-
+    def getOutputPath(prompt, default):
+        prompt_cli = prompt + f' ({default=}): '
```

Comment:
Does `=` make sense here? I've not seen it used for strings.

Reply:
Resolved? Or should I put the "default: FILENAME_HERE" back with a colon

Reply:
Well I mean, it's a part of the format string, does it have any effect here?

Reply:
It just prints the variable name and equals sign before the value :)

Reply:
I see! Didn't know about that.